sprocs/grading_jobs_insert_internal.sql

```diff
@@ -69,6 +77,18 @@ BEGIN
     WHERE
         s.id = submission_id;
 
+    IF new_gradable = FALSE THEN
+        -- If the submission is ungradable then we should stop here
```

Comment:
I would like to still insert a `grading_job` even in the case that the submission was ungradable. That would preserve the idea that `grading_jobs` correspond to attempts to grade for both internal and external questions. I guess we should add a new `gradable` column to `grading_jobs` to record whether the grading job returned `gradable=false`.